LICENSE

```diff
@@ -0,0 +1,28 @@
+BSD 3-Clause License
+
+Copyright (c) 2023, Prefix GmbH
```

Comment:
If we use prefix, then better to use `prefix.dev GmbH` as that's the actual name :)